src/test/data/JsonModuleBookStorageTest/invalidAndValidTaskModuleBook.json

```diff
@@ -3,11 +3,13 @@
     "name": "Valid Task",
     "deadline": "9482424",
     "module": "hans@example.com",
-    "description": "4th street"
+    "description": "4th street",
+    "doneStatus": "false"
   }, {
     "name": "Task With Invalid Deadline Field",
     "deadline": "948asdf2424",
     "module": "hans@example.com",
-    "description": "4th street"
+    "description": "4th street",
+    "doneStatus": "false"
   } ]
 }
```

Comment:
Here the value of property isn't change to correct format of MB3.5.

Reply:
To clarify, the module field should be changed to a proper module code?